src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -140,6 +160,7 @@ export class TimeseriesChart extends Component<
           yAccessor: s.yAccessor || DataLoader.yAccessor,
           y0Accessor: s.y0Accessor || DataLoader.y0Accessor,
           y1Accessor: s.y1Accessor || DataLoader.y1Accessor,
+          ySubDomain: this.getYdomainValues(s.yDomainScaleOptions),
```

Comment:
We have defined interface for series, it should be expanded with `ySubDomain` type.